eng/release/Scripts/GetBarId.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
```

Comment:
For my learning: Are we using BAR ID for anything other that correlation purposes anymore in dotnet-monitor?

Reply:
We use it for calling `darc gather-drop` in the AcquireBuild script and for getting the release version number. In the near future, I'll use it to get the build version number as well.